oracle-plugin/docs/Oracle-batchsink.md

```diff
@@ -30,6 +30,8 @@ You also can use the macro function ${conn(connection-name)}.
 
 **SID/Service Name:** Oracle connection point (Database name or Service name).
 
+**Role** Login role of the user when connecting to the database.
```

Comment:
is there an example we can give - like what kind of roles?

Reply:
ack